README.md

```diff
@@ -126,3 +126,21 @@ interface PlaidLinkOptions {
 ## Typescript support
 
 Typescript definitions for `react-plaid-link` are built into the npm packge.
+
+## Publishing new versions
+
+You'll need npm publishing rights for this package for your npm user.
+
+```
+npm version <version specifier>
```

Comment:
@timruffles – we actually have a simple command for publish:

```
make release-(patch|minor|major)
```

It'll handle all of these steps - and is how we have published all npm libs so far